woocommerce/wc-admin
# Add countries data store

Fixes (part of) #7782 

Adds a data store for retrieving data from the locales endpoint.

### Screenshots
<img width="439" alt="Screen Shot 2022-01-05 at 5 38 47 PM" src="https://user-images.githubusercontent.com/10561050/148300116-3fffab07-7a6c-4527-ba24-bf7644b6f2c2.png">



### Detailed test instructions:

1. Open your browser's console.
2. Enter `wp.data.select('wc/admin/countries').getLocales()`
3. You'll need to run the above at least twice (once for resolving and again once resolved to see the data).
4. Note that the locale data is shown
5. Enter `wp.data.select('wc/admin/countries').getLocale('US')` after data has been fetched
6. Note that the single locale data is shown
